news/2023/2023-01-19-gmt-apps-now-open.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Global Moderation Team applications now open!
+date: 2023-01-19 12:00:00 +0000
+---
+
+Been an active community member for a while? Felt an inner urge to chase after some osu! troublemakers? Now it's your time to shine! The Global Moderation Team is seeking new members.
+
+![](/wiki/shared/news/banners/gmt-generic.jpg)
+
+It's been over 2 years since the [last GMT application cycle](https://osu.ppy.sh/home/news/2020-10-04-global-moderation-team-applications-open), and while some things have changed, one thing remains the same — we may be looking for you!
+
+The [Global Moderation Team](https://osu.ppy.sh/groups/4) is responsible for keeping the community in a good condition, keeping it a welcoming place for everyone. Our green-coloured peers of the GMT not only help moderate the in-game chat and forums, but also engage with the mapping community among others. The members of the team can also give you assistance with some of your technical issues. The inherent goal for is to reach closer and closer to the living community.
+
+That being said, this time we will look not only for chat or forum moderators. If you feel you're a prominent figure in the osu! community and would like to help us with the community moderation side of things, be sure to shoot us an application as well!
+
+While usually we seek new members in a [process of recommendations and nominations](/wiki/People/The_Team/Global_Moderation_Team#becoming-a-moderator), we're also opening applications like this one from time to time to look for fresh blood we might not be otherwise noticing — not everyone is so outspoken and perceptible at the end of the day! While we will still continue using our classic methods, we believe that reaching out to the community directly is vastly valuable to the team's health.
+
+Before you apply, though, be sure to take a look through the list of factors regarding your application below:
+
+- **You must be at least 18 years old. This is NOT negotiable.** The GMT role means that you will have to deal with content of inappropriate nature that's not suitable for all ages.
```

Comment:
probably also worth linking that this isnt just "NOT negotiable", but even is part of the osu! TOS.
https://osu.ppy.sh/legal/en/Terms#your-responsibilities-in-volunteer-positions